src/lib/client.ts

```diff
@@ -0,0 +1,39 @@
+import { createPublicClient, createWalletClient, http } from 'viem';
+import { sepolia } from 'viem/chains';
+import { privateKeyToAccount } from 'viem/accounts';
+require('dotenv').config();
+
+let publicClient: ReturnType<typeof createPublicClient> | null = null;
```

Comment:
This is not a good way to export constants. you can just return these variables from  initialiseClients.

That way, someone using these functions and constants wont need to know how to import them individually